PowerShell/Module/Templates/Blueprints/S3EventToSQS/s3tosqs.ps1.txt

```diff
@@ -6,16 +6,17 @@
 #
 # The last item in the PowerShell pipeline will be returned as the result of the Lambda function.
 #
-# To include PowerShell modules with your Lambda function, like the AWSPowerShell.NetCore module, add a "#Requires" statement
-# indicating the module and version.
+# To include PowerShell modules with your Lambda function, like the AWS.Tools.S3 module, add a "#Requires" statement
+# indicating the module and version. If using an AWS.Tools.* module the AWS.Tools.Common module must also be a required.
```

Comment:
nit: "must also be a required." => "is also required.". Or at least drop the "a" before required. (Repeated on the other *.txt files as well)

Reply:
Fixed in all templates